cmd/jujud/updateseries/updateseries.go

```diff
@@ -103,8 +98,33 @@ func (c *UpdateSeriesCommand) SetFlags(f *gnuflag.FlagSet) {
 	f.BoolVar(&c.startAgents, "start-agents", false, "start agents if successful with update")
 }
 
+// getJujuVersion gets the version of agent
+func (c *UpdateSeriesCommand) getJujuVersion() (version.Number, error) {
+	// Get the current juju version from the machine agent
+	// conf file.
+	agentConf := agent.NewAgentConf(c.dataDir)
+	if err := agentConf.ReadConfig(c.machineAgent); err != nil {
+		return version.Number{}, err
+	}
+	config := agentConf.CurrentConfig()
+	if config == nil {
+		return version.Number{}, errors.Errorf("%s agent conf is not found", c.machineAgent)
+	}
+	return config.UpgradedToVersion(), nil
+}
+
 func (c *UpdateSeriesCommand) Run(ctx *cmd.Context) error {
-	if err := c.findAgents(ctx); err != nil {
+	var err error
```

Comment:
These don't all need to be vars, eg errSysdServiceNames, errSymServiceNames. They can just be assigned to directly where they're used.
For the couple that do need to be vars, use grouping,

var (
  err error
  errAgentNames []string
)